### PR TITLE
inherit from BaseCommand instead of LabelCommand if handle_label is n…

### PR DESCRIPTION
**Pull request by nickpell**
…ot defined

Needed for compatibility with django 1.8

@calellowitz

### CODE REVIEW

File: corehq/ex-submodules/toggle/management/commands/make_toggle.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
-from django.core.management.base import LabelCommand, CommandError
+from django.core.management.base import BaseCommand, CommandError
 from ...models import Toggle
 
-class Command(LabelCommand):
+class Command(BaseCommand):
```
Comment (dimagimon) [inline corehq/ex-submodules/toggle/management/commands/make_toggle.py]: E302 expected 2 blank lines, found 1

File: hqscripts/management/commands/clean_couchlog.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
-from django.core.management.base import LabelCommand
+from django.core.management.base import BaseCommand
 from couchlog.models import ExceptionRecord
-class Command(LabelCommand):
+class Command(BaseCommand):
```
Comment (dimagimon) [inline hqscripts/management/commands/clean_couchlog.py]: E302 expected 2 blank lines, found 0

**Comment by calellowitz** [comment]
👍 on ✅

**Comment by nickpell** [comment]
@calellowitz

**Comment by nickpell** [comment]
✅

**Pull request merged**